gcm_filters/filter.py

```diff
@@ -282,6 +285,38 @@ def __post_init__(self):
             )
         self.grid_ds = xr.Dataset({name: da for name, da in self.grid_vars.items()})
 
+    def plot_shape(self):
```

Comment:
```suggestion
    def plot_shape(self, ax=None):
```

Let's add an option to pass an existing axis object, in case the user wants to put the plot into a broader figure.